packages/blade/src/components/Link/BaseLink/BaseLink.tsx

```diff
@@ -46,7 +46,7 @@ type BaseLinkCommonProps = {
    *
    * @default medium
    */
-  size?: 'small' | 'medium' | 'large';
+  size?: 'xsmall' | 'small' | 'medium' | 'large';
```

Comment:
@saurabhdaware shouldn't we abstract these sizes now as we get ad-hoc requests? for eg: this can probably go as `BaseTextSizes` and we can use that everywhere so things remain in sync?

Reply:
Oh yup we can do that. For this PR, I removed the repeatation from `Link` and `BaseLink` and just added `BaseLinkProps['size']` there.

Reply:
created issue #1264 to track